### PR TITLE
feat(addon-editor): add ability to color cells

### LINKED ISSUE #756: [FEATURE] `Editor` add ability to color cells
# 🚀 Feature request

Now it is impossible to color the table cells in the Editor, let's add this feature

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #756 

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: projects/addon-editor/components/editor-new/editor-new.component.ts [hunk 1]
```diff
@@ -117,7 +116,12 @@ export class TuiEditorNewComponent
     }
 
     onMouseDown(event: MouseEvent) {
-        if (this.editor?.isFocused) {
```
Comment (waterplea) [inline projects/addon-editor/components/editor-new/editor-new.component.ts:120]: Could you clarify this change?

Comment (vladimirpotekhin) [reply projects/addon-editor/components/editor-new/editor-new.component.ts:120]: Yes, it is fix for issue with focus. Previously, focus was lost when clicking on a disabled button in a toolbar or wrapper padding.

Comment (waterplea) [reply projects/addon-editor/components/editor-new/editor-new.component.ts:120]: Maybe preventing default on mousedown would be preferable? On the items that you don't want to drag the focus away.

Comment (vladimirpotekhin) [reply projects/addon-editor/components/editor-new/editor-new.component.ts]: I set preventing mousedown on toolbar and move padding to prosemirror container. This is much better, thanks!

**Review by nsbarsukov — DISMISSED** [review]
@vladimirpotekhin 
Let's fill the color of table' cell on [demo example](https://taiga-ui.dev/components/editor-new#basic) so the tests check new feature?

**Review by waterplea — APPROVED** [review]

**Review by waterplea — DISMISSED** [review]

**Review by waterplea — APPROVED** [review]

**Comment by nsbarsukov** [comment]
@vladimirpotekhin 
I have seen that you added commit with initial color filling.
But I opened [preview of demo](https://github.com/TinkoffCreditSystems/taiga-ui/pull/766#issuecomment-926499140) and don't see it ;(

**Comment by nsbarsukov** [comment]
it is a problem of preview bot :(
I launched the branch locally and everything is fine

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**